analyzer/queries/queries.go

```diff
@@ -601,10 +601,13 @@ var (
 
 	RuntimeEVMNFTInsert = `
```

Comment:
```suggestion
	RuntimeEVMNFTUpsert = `
```

Reply:
):

Reply:
no on this. there's a lot of `...Insert` queries that define an ON CONFLICT. really we need better terminology for "the thing the block analyzer does" and "the thing the other analyzer does"

Reply:
actually nah changing it is fine. most of the insert on conflict's are DO NOTHING

Reply:
:+1: if they're not `DO NOTHING`, I'm in favor of renaming them too. Consistency is always nice. But regardless, I find the "upsert" in the name helpful when reading analyzer code; sometimes that clue is enough to guess/remember what the SQL query does, without jumping to the definition.